### PR TITLE
Build sh consistency

### PR DESCRIPTION
**Pull request by zxcv05**
Tweaks `src/build.sh` and `src/bf2path/build.sh` to be more consistent with eachother
Also changes the dockerfile and build.sh to no longer build a new container image every new commit

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -6,30 +6,29 @@ log() {
...
     log "[*] Building container"
     log ""
 
-    docker-compose build --build-arg COMMIT="$(git log -1 --format=%h)"
+    docker-compose build
 
     log ""
 fi
 
+if [ -z "$1" ]; then CONTAINER_EXEC="time bash -e /code/start-build.sh";
```
Comment (IOKG04) [inline build.sh]: For consistency, may i suggest making `build` a valid option here too?

**Review by IOKG04 — CHANGES_REQUESTED** [review]

**Comment by IOKG04** [comment]
mind if i ask for the reasoning behind the reordering of the install commands in the dockerfile?  
just so i can do it right the next time

File: build.sh (hunk 1 above)
Comment (zxcv05) [reply build.sh]: on it

**Comment by zxcv05** [comment]
> mind if i ask for the reasoning behind the reordering of the install commands in the dockerfile? just so i can do it right the next time

ocd nice looking ascending order alphabetical :yum:

File: build.sh (hunk 1 above)
Comment (zxcv05) [reply build.sh]: done

**Comment by IOKG04** [comment]
> > mind if i ask for the reasoning behind the reordering of the install commands in the dockerfile? just so i can do it right the next time
> 
> ocd nice looking ascending order alphabetical :yum: 

i highly wonder which alphabet u use where z comes before b, but okay

**Comment by zxcv05** [comment]
> > > mind if i ask for the reasoning behind the reordering of the install commands in the dockerfile? just so i can do it right the next time
> > 
> > 
> > ocd nice looking ascending order alphabetical 😋
> 
> i highly wonder which alphabet u use where z comes before b, but okay

ascending order THEN alphabetical

**Review by IOKG04 — APPROVED** [review]
Looks good now

**Comment by IOKG04** [comment]
> > > > mind if i ask for the reasoning behind the reordering of the install commands in the dockerfile? just so i can do it right the next time
> > > 
> > > 
> > > ocd nice looking ascending order alphabetical 😋
> > 
> > i highly wonder which alphabet u use where z comes before b, but okay
> 
> ascending order THEN alphabetical

couldve communicated that more clearly...  
anyway get it now

**Pull request merged**